Formula/stone.rb

```diff
@@ -15,7 +15,7 @@ class Stone < Formula
   option "with-ssl", "SSL support"
 
   def install
-    target = (build.with? "ssl") ? "macosx-ssl" : "macosx"
+    target = build.with?("ssl") ? "macosx-ssl" : "macosx"
```

Comment:
Unrelated but I'd probably bet money that this is trying to use the system OpenSSL.